virttest/qemu_devices/qdevices.py

```diff
@@ -1476,6 +1476,13 @@ def _insert(self, device, addr):
         self.bus[addr] = device
         return []
 
+    def hotplug(self, device):
+        """
+        Hotplug device
+        :param device: QDevice object
+        """
+        device.set_param["bus"] = self.busid
```

Comment:
I'm not sure about this. The `QSparseBus` is a generic bus implementation and some devices might not use the `bus` property. Anyway let's go with it for now and we can fix it in case some problems occur.

Reply:
Actually functional test uncovered something... This should be `device.set_param("bus", self.busid)`.